lib/drawers/publish.vue

```diff
@@ -237,6 +238,14 @@
     }
   }
 
+  function isInThePast(dateValue, timeValue) {
```

Comment:
Can we add code comments that make it clear what `dateValue` and `timeValue` are?

Reply:
good point